airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricQueries.java

```diff
@@ -153,43 +153,63 @@ public static List<Pair<JobStatus, Double>> overallJobRuntimeForTerminalJobsInLa
    * are not matching with the number of expected runs according to the schedule settings. Refer to
    * playbook for detailed discussion.
    */
-  public static Long numOfJobsNotRunningOnSchedule(final DSLContext ctx) {
+  public static Long percentOfJobsNotRunningOnSchedule(final DSLContext ctx) {
     final var countField = "cnt";
     final var query = """
-                      SELECT count(1) as cnt FROM ((
-                      	SELECT
+                      with abnormal_sync_jobs as
```

Comment:
can we write a test for this? This is a complex query so it's important to know we are getting this right

Reply:
sure - i didn't check we actually have tests for these queries. Will do.